l10n_us_check_writing_address/report/print_check.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.osv import osv
+from odoo.report import report_sxw
+from odoo import api, fields, models, _
+
+LINE_FILLER = '*'
```

Comment:
It would be nicer if these were contained within the `ReportPrintCheck` class IMO. Even better though would be system params or company settings.